style/header.css

```
@@ -0,0 +1,40 @@
+.header{
+    background-color: var(--main-bg-color);
+    background-size: cover;
+    gap: 154px;
+    height: 1023px;
```

I suppose this is the height of screen on your device. To make an image full-sized not depending on particular device size, consider using `vw` and `vh`
```suggestion
    height: 100vh;
```